starter-code/routes/celebrities.routes.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const router  = express.Router();
+
+const CelebrityModel = require('../models/celebrity-model')
+
+/* GET home page */
+router.get('/celebrities', (req, res, next) => {
+  console.log("Hello")
+    CelebrityModel.find()
+      .then((celebrities) => {
+        console.log(celebrities)
+
+        res.render('celebrities/index.hbs', {celebrities})
+      })
+      .catch((err) => {
+        console.log('Oh-no!', err)
+        next()
+      })
+  });
+
+  router.get('/celebrities/:id', (req, res, next) => {
+    let myId = req.params.id
```

Comment:
you can use the same Id variable name as the param. In fact using destructure to get the variable from the param obj is good practice.
`const { id } = req.params`